src/actors/actor.go

```diff
@@ -18,34 +20,59 @@ import cid "github.com/ipfs/go-cid"
 //   Yes, a user may submit N separate messages with increasing
 //   sequence number, causing them to execute in order.
 //
-type CallSeqNum UVarint
+type CallSeqNum util.UVarint
 
 // Code is a serialized object that contains the code for an Actor.
 // Until we accept external user-provided contracts, this is the
 // serialized code for the actor in the Filecoin Specification.
-type Code Bytes
+type Code abi.Bytes
+
+type ActorSystemStateCID cid.Cid
 
 // Actor is a base computation object in the Filecoin VM. Similar
 // to Actors in the Actor Model (programming), or Objects in Object-
 // Oriented Programming, or Ethereum Contracts in the EVM.
 //
 // ActorState represents the on-chain storage all actors keep.
 type ActorState struct {
-    // Identifies the code this actor executes.
-    CodeID      abi.ActorCodeID
-    // CID of the root of optional actor-specific sub-state.
-    State       ActorSubstateCID
-    // Balance of tokens held by this actor.
-    Balance     abi.TokenAmount
-    // Expected sequence number of the next message sent by this actor.
-    // Initially zero, incremented when an account actor originates a top-level message.
-    // Always zero for other abi.
-    CallSeqNum
+	// Identifies the code this actor executes.
+	CodeID_ abi.ActorCodeID
+	// CID of the root of optional actor-specific sub-state.
+	State_ ActorSubstateCID
+	// Balance of tokens held by this actor.
+	Balance_ abi.TokenAmount
+	// Expected sequence number of the next message sent by this actor.
+	// Initially zero, incremented when an account actor originates a top-level message.
+	// Always zero for other abi.
+	CallSeqNum_ CallSeqNum
+}
```

Comment:
We don't need to move this structure, the actors can't see it! ActorSystemStateCID likewise.

It's probably best to restore this to the location you moved it from. It can remain an .id file in that case, at your option.